### ISSUE TITLE
groups.yml get wiped out every time after server restarted

### ISSUE THREAD

**Issue by k-jiang** [NLthijs48/AreaShop#299]
#### Current behavior
`groups.yml` get wiped out every single time after server restarted. Even though I could load the groups.yml by `/as reload` command normally.

#### Steps to reproduce (for bugs)

1. put your groups.yml into you plugin folder
2. `/as reload` and it works
3. restart your server
4. `groups.yml` is wiped out, file size becomes zero, gg, have to manually repeat step 1 to make it works again

#### Your environment (for bugs)
* Server platform (Bukkit, Spigot, etc.): Spigot
* Server version: 1.11.2 build 1210
* AreaShop version: 2.4.0
* WorldEdit version: 6.15
* WorldGuard version: 6.2
* Vault version: 1.5.6

#### Your config files (for bugs)

`groups.yml`:
```
xinshoucun-shop:
  name: xinshoucun-shop
  priority: 100
  rent:
    price: 300
    maxRentTime: '30 days'
  general:
    signProfile:
      forrent:
        line1: '&2&l[商店出租]'
        line2: '编号: %region%'
        line3: '%duration%'
        line4: '%price%'
      rented:
        line1: '&4&l[已出租]'
        line2: '编号: %region%'
        line3: '商户: %player%'
        line4: '到期: %untilshort%'
      forsale:
        line1: '&2&l[商店出售]'
        line2: '编号: %region%'
        line3: ''
        line4: '%price%'
      resell:
        line1: '&9&l[商店转让]'
        line2: '编号: %region%'
        line3: '卖家: %resellprice%'
        line4: '&8%player%'
      sold:
        line1: '&4&l[已出售]'
        line2: '编号: %region%'
        line3: '商户: %player%'
        line4: ''
    schematicProfile:
      created:
        save: ''
        restore: ''
      deleted:
        save: ''
        restore: ''
      rented:
        save: ''
        restore: ''
      unrented:
        save: ''
        restore: 'rent-xinshoucun-shop-xx'
      bought:
        save: ''
        restore: ''
      sold:
        save: ''
        restore: ''
      resell:
        save: ''
        restore: ''
    enableRestore: true
  regions:
  - pri-shop-01
  - pri-shop-02
  - pri-shop-03
  - pri-shop-04
  - pri-shop-05
  - pri-shop-06
  - pri-shop-07
  - pri-shop-08
  - pri-shop-09
  - pri-shop-10
  - pri-shop-11
  - pri-shop-12
  - pri-shop-13
```

**Comment by k-jiang** [comment]
Correction: the `groups.yml` is not wiped out by restart but somehow during server running. Perhaps it is due to some sort of auto-saving feature in AreaShop?

**Comment by NLthijs48** [comment]
Every time you change something with groups ingame (using `/as groupadd` or `/as groupdel`) AreaShop will remember that the `groups.yml` file needs to be saved. Every 15 minutes AreaShop will check if the file needs to be saved and actually save it to disk. Saving also happens at shutting down the server and when using `/as reload` (only if changes have been made still).

I also copied your configuration in my `groups.yml` file to test if loading and saving the Chinese? characters works correctly, which it does for me. So when saving it makes the file completely empty for you? Is there anything in the console when AreaShop shuts down or starts up? Because one thing it might be is that AraeShop does not read the file correctly, and ends up with an empty `groups.yml` file. You also might want to check if the text editor you use saving the file as UTF-8, because that is how AreaShop is trying to load the file (UTF-8 support all characters, while other encodings only support some languages).

Let me know if this information helps and if you know anything we can do so that I can reproduce the issue.

**Comment by k-jiang** [comment]
All files are saved in UTF-8 (without BOM) encoding. No error shows up during server startup, shutdown or running.

The problem is, every time when I shutdown my server and start it up again, AreaShop seems not loading the `groups.yml` file at all because all the setting in this files are not take in effects. At this point the `groups.yml` is still fine. Then after a period of time the `groups.yml` becomes empty. I tested it on my PC and that period of time is exactly 15min as you said.

I also tried the latest build, but still no luck.
